examples/src/main/java/ai/djl/examples/inference/ObjectDetectionWithTensorflowSavedModel.java

```diff
@@ -0,0 +1,250 @@
+ * href="https://github.com/awslabs/djl/blob/master/examples/docs/object_detection_with_tensorflow_saved_model.md">doc</a>
+ * for information about this example.
+ */
+public final class ObjectDetectionWithTensorflowSavedModel {
+
+    private static final Logger logger =
+            LoggerFactory.getLogger(ObjectDetectionWithTensorflowSavedModel.class);
+
+    private ObjectDetectionWithTensorflowSavedModel() {}
+
+    public static void main(String[] args) throws IOException, ModelException, TranslateException {
+        DetectedObjects detection = ObjectDetectionWithTensorflowSavedModel.predict();
+        logger.info("{}", detection);
+    }
+
+    public static DetectedObjects predict() throws IOException, ModelException, TranslateException {
+        if (!"TensorFlow".equals(Engine.getInstance().getEngineName())) {
+            throw new UnsupportedOperationException("This test only for TensorFlow");
+        }
+
+        Path imageFile = Paths.get("src/test/resources/dog_bike_car.jpg");
+        Image img = ImageFactory.getInstance().fromFile(imageFile);
+
+        String modelUrl =
+                "http://download.tensorflow.org/models/object_detection/tf2/20200711/ssd_mobilenet_v2_320x320_coco17_tpu-8.tar.gz";
+
+        Criteria<Image, DetectedObjects> criteria =
+                Criteria.builder()
+                        .optApplication(Application.CV.OBJECT_DETECTION)
+                        .setTypes(Image.class, DetectedObjects.class)
+                        .optModelUrls(modelUrl)
+                        // saved_model.pb file is in the subfolder of the model archive file
+                        .optModelName("ssd_mobilenet_v2_320x320_coco17_tpu-8/saved_model")
+                        .optTranslator(new MyTranslator())
+                        .optProgress(new ProgressBar())
+                        .build();
+
+        try (ZooModel<Image, DetectedObjects> model = ModelZoo.loadModel(criteria);
+                Predictor<Image, DetectedObjects> predictor = model.newPredictor()) {
+            DetectedObjects detection = predictor.predict(img);
+            saveBoundingBoxImage(img, detection);
+            return detection;
+        }
+    }
+
+    private static void saveBoundingBoxImage(Image img, DetectedObjects detection)
+            throws IOException {
+        Path outputDir = Paths.get("build/output");
+        Files.createDirectories(outputDir);
+
+        // Make image copy with alpha channel because original image was jpg
+        Image newImage = img.duplicate(Image.Type.TYPE_INT_ARGB);
+        newImage.drawBoundingBoxes(detection);
+
+        Path imagePath = outputDir.resolve("detected-tensorflow-model-dog_bike_car.png");
+        // OpenJDK can't save jpg with alpha channel
+        newImage.save(Files.newOutputStream(imagePath), "png");
+        logger.info("Detected objects image has been saved in: {}", imagePath);
+    }
+
+    static Map<Integer, String> loadSynset() throws IOException {
```

Comment:
FYI: DJL defines synset as List is just provide a convenient way to create ImageTranslator, you can load your label mapping in whatever way you want in your custom Translator